Kubernetes 1.21.2

helgi opened the pull request:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#v1212

wadells commented:
@bernardjkim: One of the changes mentioned in https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#v1212 is:

> Fixed garbage collection of dangling VolumeAttachments for PersistentVolumes migrated to CSI on startup of kube-controller-manager. (kubernetes/kubernetes#102176, @timebertt) [SIG Apps and Storage]

Is this potentially related to the issue we saw in staging?

helgi committed:
Kubernetes 1.21.2

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#v1212
